README.asciidoc

```diff
@@ -179,8 +179,21 @@ some steps.
   # vagrant plugin install vagrant-bindfs
 --------------------------------------
 
-This is all you need to prepare initally to set up the vagrant environment,
-now you can simply start the virtual machine with +vagrant up+ and start
+* If you plan to use +libvirt+ as provisioner make sure you have the 
+  libvirt-plugin installed:
+
+--------------------------------------
+  # vagrant plugin install vagrant-libvirt
+--------------------------------------
+
+This is all you need to prepare initally to set up the vagrant environment.
```

Comment:
I could argue that it's actually kind of correct as the other step is not vagrant related but I'll reword/reorder it.